tdrs-frontend/src/selectors/auth.js

```diff
@@ -6,6 +6,25 @@ export const selectUser = (state) => state.auth.user || null
 export const selectUserAccountApprovalStatus = (state) =>
   selectUser(state)?.['account_approval_status']
 
+export const selectUserRoles = (state) => selectUser(state)?.['roles'] || []
+
+export const selectPrimaryUserRole = (state) => {
+  const roles = selectUserRoles(state)
+  if (roles.length > 0) {
+    return roles[0]
+  }
+  return null
+}
+
+export const selectUserPermissions = (state) => {
+  const roles = selectUserRoles(state)
+  let permissions = []
+  roles.forEach((role) => {
+    permissions = [...permissions, role['permissions']]
+  })
+  return permissions
+}
```

Comment:
this is (hopefully) an improvement to avoid us getting in similar situations in the future. We should opt to standardize the selectors as much as possible in order to ensure the same implementation across the app.